### DISCUSSION TITLE
Getting a program to work in dosbox-x

### DISCUSSION THREAD

**Discussion by grstatdoc-gr** [joncampbell123/dosbox-x#4310] [Q&A]
Hi...
I am wanting to run The Automated Scorebook for Baseball in DosBox-X. So far, I have the needed folders in DosBox-X and while the program runs, I am having some problems accessing files. So, I have the program directory mounted c:\tasbs, but I'm wondering if the subdirectories would be automatically accessible.
I'll mount screenshots if needed a bit later, since I'm not on the laptop that I have DosBox-X installed right now.

**Comment by rderooy** [comment]
You seem to be using a Windows PC. In that case it is probably best if you create a directory on the Windows host such as C:\dosroot or C:\Users\MyUser\dosroot

Then in that directory create (or move) your "tasbs" directory, such that you have something like C:\dosroot\tasbs on your Windows host. Now in DOSBox-X type `mount c c:\dosroot` and the "dosroot" directory will become your virtual C: drive in DOSBox-X.

If you now type `DIR` in DOSBox-X you should see the "tasbs" directory.

**Comment by grstatdoc-gr** [comment]
Thanks for the tip!
Yes, I am using a Windows 11 laptop. I'll check it out a bit later.
Mike Dean

**Reply by grstatdoc-gr** [reply]
Yes, I did follow the steps and and I can see the game schedule and load game files. Thanks!!!!
Now, my next step is to figure out how to run the Windows reports, but that involves a different step.

**Comment by grstatdoc-gr** [comment]
Well, after some trials, errors and tribulations, I got a Windows 98 guest running in DOSBox-X. Are there guides to getting TAS to run in that guest system I can consult, as I'm having having problems figuring out how to get the guest to see the DOSBox-X folder to access the program?
(I wound up moving the files to the DOSBox-X folder, since I had the understanding that I needed to mount that to install Win98, but I didn't need to...)

**Comment by rderooy** [comment]
On the wiki there is a page amount managing image files, that should help you with transferring files into a disk image.
http://dosbox-x.com/wiki/Guide%3AManaging-image-files-in-DOSBox%E2%80%90X

**Comment by grstatdoc-gr** [comment]
I am looking at this sequence in the wiki:

IMGMOUNT C hdd.img
MOUNT D .
COPY D:\UNZIP.EXE C:\DOS

I am not sure how the mount D . command works (I'm assuming I've started
from the Z: prompt before running these commands)...
So does the "." mean that the contents of C:\DOSBox-X\ is referred to here
(the files I need are two subdirectories there)?
Mike Dean

**Reply by rderooy** [reply]
`.` is a reference to the CWD (Current Working Directory) on the host. You can replace it with any host path.

**Comment by grstatdoc-gr** [comment]
Thanks, I'll check it out on Monday and post results.
Mike Dean

**Reply by grstatdoc-gr** [reply]
More progress...
I got the needed directories on the hdd.img file and am running Win98 OK (fussing around with the best display size ... desktop is too small; fullscreen is too distorted, and not sure where to put the aspect=true option

> #    fullresolution: What resolution to use for fullscreen: original, desktop or a fixed size (e.g. 1024x768).
> #                        Using your monitor's native resolution with aspect=true might give the best results.
> 

I'm also struggling to find a way of, while in win98, sending files outside of it (without having to resort to copy or xcopy) ... I'm not sure if a network folder can be created (as I have between the host system and VirtualBox, by comparison)

**Comment by rderooy** [comment]
You can set a resolution in Win98 itself that suits your needs.

As to accessing a network folder from within Win98, yes that is in theory possible. This is a pretty advanced topic for which you need to setup networking in both DOSBox-X and in Win98.

Once that is working, you can **try** to connect to a SMB/CIFS server on your network. **BUT** Win98 only supports the very early SMBv1 protocol, which any modern SMB/CIFS server will no longer have enabled by default. And some don't even support any way of enabling SMBv1 any longer.
